.github/workflows/build.yml

```diff
@@ -52,3 +52,20 @@ jobs:
         kubectl apply -n argo -f manifests/mpi-operator.yaml
 
         scripts/integration_tests.sh
+  go-unittest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.1' # The Go version to download (if necessary) and use.
```

Comment:
This is different than what's in `go 1.15`?

Reply:
Will keep them constant later.

Reply:
Thanks, will keep them consistent in the later commit.